_layouts/default.html

```diff
@@ -7,8 +7,20 @@
 
     {%- include header.html -%}
 
+    <div class="vide-target" data-vide-bg="/assets/images/aero.min.mp4" data-vide-options="posterType: mp4, muted: true, position: 0% 0%, resizing: false"></div>
+
     <main class="page-content" aria-label="Content">
+
       <div class="wrapper">
+
+        <p style="text-align: center; padding-top: 20px;">
+          <img src="/assets/images/logos/aerosec.svg" alt="logo" />
+        </p>
+
+        <h1 style="text-align: center;">
+          AEROSEC: The Aviation Cyber Security Group
+        </h1>
+
```

Comment:
I'm a little unsure if this title info belongs outside the `main` or at least outside of the bit with `aria-label="Content"`, but since I don't really know how tools deal with the various aria labels, I'm happy enough leaving it. (I've been making an effort to make my web pages be comprehensible to people using screen readers, but honestly, it's hard to do more than guess without one.)